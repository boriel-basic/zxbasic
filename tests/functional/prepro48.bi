#define X A(0)
#define A(x) x

#if X(A)
print 1
' This should return no code once filtered by preprocessor
#endif



