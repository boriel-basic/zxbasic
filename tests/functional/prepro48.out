#line 1 "prepro48.bi"



#line 8 "prepro48.bi"



